doc/roadmap.rst

```diff
@@ -138,17 +138,21 @@ bottom.
    * Grid search and cross validation are not applicable to most clustering
      tasks. Stability-based selection is more relevant.
 
+#. Better support for manual and automatic pipeline building
+
+   * Easier way to construct complex pipelines and valid search spaces
+     :issue:`7608` :issue:`5082` :issue:`8243`
+   * provide search ranges for common estimators??
+   * cf. `searchgrid <https://searchgrid.readthedocs.io/en/latest/>`_
+
 #. Improved tracking of fitting
 
    * Verbose is not very friendly and should use a standard logging library
-     :issue:`6929`
+     :issue:`6929`, :issue:`78`
    * Callbacks or a similar system would facilitate logging and early stopping
 
 #. Distributed parallelism
-
-   * Joblib can now plug onto several backends, some of them can distribute the
-     computation across computers
-   * However, we want to stay high level in scikit-learn
+   * Accept data which complies with ``__array_function__``
```

Comment:
I'm confused, isn't https://github.com/scikit-learn/scikit-learn/pull/15332 doing the exact opposite (i.e. not rely on ``__array_function__``)?

Reply:
You mean Andy's PR? The idea is for now actively not to accept it, instead of giving odd errors if the user passes them, and then work towards possibly supporting them later.

Reply:
lol yeah sorry I meant https://github.com/scikit-learn/scikit-learn/pull/14702